akka-persistence-query/src/main/scala/akka/persistence/query/EventEnvelope.scala

```diff
@@ -59,15 +84,19 @@ final class EventEnvelope(
   }
```

Comment:
we don't include meta in `equals` for backwards compatibility (same as when we added timestamp)